scan/templates/txs_pending/list.html

```diff
@@ -23,5 +24,12 @@ <h5 class="p-2">Pending Transactions</h5>
       </div>
     </div>
   </div>
+</div>
+
+<script>
+  setInterval(function () {
+    $('#update_pending').load(location.href + ' #update_pending');
+  }, 10000); //update every X seconds
```

Comment:
miliseconds?

Reply:
lol, yes..  just notes for us... ill change description to ms.